sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m

```diff
@@ -0,0 +1,172 @@
+//
+//  OOReactSkinBridgeModuleMain.m
+//  OoyalaSkinSDK
+//
+//  Created by Maksim Kupetskii on 8/14/18.
+//  Copyright © 2018 ooyala. All rights reserved.
+//
+
+#import "OOReactSkinBridgeModuleMain.h"
+#import "NSMutableDictionary+Utils.h"
+
+#import <OoyalaSDK/OODebugMode.h>
+#import <OoyalaSDK/OOOoyalaPlayer.h>
+
+@interface OOReactSkinBridgeModuleMain ()
+
+@property (nonatomic, weak, nullable) id<OOReactSkinModelDelegate> skinModelDelegate;
+
+@end
+
+
+@implementation OOReactSkinBridgeModuleMain
+
+@synthesize bridge;
+
+RCT_EXPORT_MODULE(OOReactSkinBridgeModuleMain);
+
+- (dispatch_queue_t)methodQueue {
+  return dispatch_get_main_queue();
+}
+
+- (OOReactSkinBridgeModuleType)moduleType {
+  return OOReactSkinBridgeModuleTypeMain;
+}
+
+- (void)setSkinViewDelegate:(nonnull id<OOReactSkinModelDelegate>)delegate {
+  _skinModelDelegate = delegate;
+}
+
+#pragma mark - Constants
+#pragma mark Buttons
+static NSString *playPauseButtonName    = @"PlayPause";
+static NSString *playButtonName         = @"Play";
+static NSString *rewindButtonName       = @"rewind";
+static NSString *socialShareButtonName  = @"SocialShare";
+static NSString *fullscreenButtonName   = @"Fullscreen";
+static NSString *learnMoreButtonName    = @"LearnMore";
+static NSString *skipButtonName         = @"Skip";
+static NSString *moreOptionButtonName   = @"More";
+static NSString *upNextDismiss          = @"upNextDismiss";
+static NSString *upNextClick            = @"upNextClick";
+static NSString *adIconButtonName       = @"Icon";
+static NSString *adOverlayButtonName    = @"Overlay";
+static NSString *stereoscopicButtonName = @"stereoscopic";
```

Comment:
Why do some of these start with a capital letter and some don't?  Is this just inconsistency with how they were declared elsewhere in the code?

Reply:
It's all in pure RN `.js` files. It's possible to do, but it will affect Android SDK as well.
I guess it's better to do as a separate task.